src/components/common/Extensions/SpecificSidePanel/SpecificSidePanel.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC, Fragment } from 'react';
+import { useIntl } from 'react-intl';
+import { SpecificSidePanelProps } from './types';
+import styles from './SpecificSidePanel.module.css';
+import ExtensionStatusBadge from '../ExtensionStatusBadge-new/ExtensionStatusBadge';
+import Permissions from './partials/Permissions';
+import DateInstalled from './partials/DateInstalled';
+import InstaledBy from './partials/InstaledBy';
+import Version from './partials/Version';
+import ContractAddress from './partials/ContractAddress';
+import Developer from './partials/Developer';
+
+const displayName = 'common.Extensions.SpecificSidePanel';
+
+const SpecificSidePanel: FC<SpecificSidePanelProps> = ({ types, sidepanelData }) => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <div className="sm:w-[23.75rem] md:w-[17.625rem] lg:w-[20.4375rem] bg-base-white">
+      <h3 className="font-semibold text-lg text-gray-900 pb-[1.375rem]">
+        {formatMessage({ id: 'specific.side.panel.title' })}
+      </h3>
+      {sidepanelData.map(
+        ({ id, dateInstalled, instaledBy, statusType, versionInstalled, contractAddress, developer, permissions }) => (
+          <Fragment key={id}>
+            <div className={styles.panelRow}>
+              <div className={styles.panelTitle}>{statusType.title}</div>
+              <div className="w-[50%] justify-start flex flex-row">
+                {types?.map((type) => (
+                  <div className=" mr-1" key={type}>
+                    <ExtensionStatusBadge mode={type} text={type} />
+                  </div>
+                ))}
+              </div>
+            </div>
```

Comment:
Maybe use enum to save status types?
And why are you comparing status to type[0]? This part of the code is hard to understand.

Reply:
I use union of strings so enum is not necessary, fixed that: types?.[0]